addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -1835,7 +1840,7 @@ QUnit.module("Views", (hooks) => {
         await click(quickCreate, ".o_field_widget[name=state] .o_priority_star:first-child");
         def = makeDeferred();
         await validateRecord();
-        assert.containsN(target, ".o_kanban_group:first-child .o_kanban_record", 1);
+        assert.containsN(target, ".o_kanban_group:first-child .o_kanban_record", 2);
```

Comment:
interesting... Does it mean that the quick create form still contains the new record data, and that that record is already displayed in the column as well?